src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -447,18 +447,21 @@ else if (conf.disk_access_mode == Config.DiskAccessMode.mmap_index_only)
         }
 
         /* Thread per pool */
-        if (conf.concurrent_reads < 2)
+        if (conf.concurrent_reads < 0)
         {
-            throw new ConfigurationException("concurrent_reads must be at least 2, but was " + conf.concurrent_reads, false);
+            throw new ConfigurationException("Concurrent reads must be non-negative", false);
         }
 
-        if (conf.concurrent_writes < 2 && System.getProperty("cassandra.test.fail_mv_locks_count", "").isEmpty())
+        if (conf.concurrent_writes < 0 && System.getProperty("cassandra.test.fail_mv_locks_count", "").isEmpty())
```

Comment:
I'd leave it as it is, people might still use it for manual testing purposes and it is not instrumental to our goals here.

Reply:
I'm not sure I understand, whereas before tests might have been able to set `concurrent_writes` to 1 for testing, now they can do so without checking that test property. In no case would tests need `concurrent_writes` set to a negative value.

If I missed something and you insist on keeping it, that's fine, but let's add a comment explaining the reason, because from looking at the code, the reason for checking the additional property is not clear.

Reply:
Oh sorry I misunderstood you at first. You're obviously right, let me remove that.